app/Models/BoardThread.php

```diff
@@ -51,19 +51,19 @@ class BoardThread extends Model
 
     protected $guarded = [];
 
-    public function cat(){
+    public function cat() {
         return $this->belongsTo('App\Models\BoardCat', 'cat_id', 'id');
     }
 
-    public function posts(){
+    public function posts() {
         return $this->hasMany('App\Models\BoardPost', 'thread_id', 'id');
     }
 
-    public function user(){
+    public function user() {
```

Comment:
__[code_sniffer]__
* Opening brace should be on a new line

<!-- Comment by [SideCI](https://sideci.com) -->